src/Command/Diff.php

```diff
@@ -86,25 +86,21 @@ protected function execute(InputInterface $input, OutputInterface $output)
         }
 
         $handler = new DiffHandler($envName, $cwd, $output->isDecorated());
+        $errorIo = $this->getErrIo($input, $output);
```

Comment:
Sure. But I expected you to update it if the naming was not conform to your own expectations ^^